### ISSUE TITLE
Implement drop

### ISSUE THREAD

**Issue by robb** [robb/Underscore.m#29]
``` objective-c
Underscore.drop(@[ @1, @2, @3 ], 1); // 2, 3
```

_Edit:_ `take` totally is `head` anyways

**Comment by ecstasy2** [comment]
Can't you do that using filter?

**Comment by robb** [comment]
Actually, it's not possible to do that only with `filter`. You need access to the element's index whereas `filter` only passes the element to its block.

**Comment by robb** [comment]
I totally failed here, `take` is already implemented by `head`. That being said, something like underscore.js' initial could by useful:

``` objective-c
Underscore.initial(@[ @1, @2, @3 ], 1); // 1, 2
```

essentially a `dropRight`

**Comment by robb** [comment]
Fixed by @vascoorey in #42
